cray_product_catalog/schema/schema.yaml

```
@@ -147,4 +151,26 @@ definitions:
         description: The type of repository
         type: string
         enum:
-          - group
\ No newline at end of file
+          - group
+  S3Repository:
+    description: An array of artifact repository.
```

If this is an array of multiple repositories, shouldn't the name be plural, like with `LoftsmanManifests`?
```suggestion
  S3Repositories:
    description: An array of artifact repositories.
```

---

Incorporated suggested change